deployment/network-operator/templates/admission_controller.yaml

```diff
@@ -0,0 +1,127 @@
+---
+{{- if .Values.admissionController.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-webhook-service
+  namespace: {{ .Release.Namespace }}
+spec:
+  ports:
+  - port: 443
+    protocol: TCP
+    targetPort: 9443
+  selector:
+    control-plane: controller-manager
+{{- end }}
+---
+{{- if .Values.admissionController.useCertManager }}
```

Comment:
Do we need to add an AND with  `.Values.admissionController.enabled`

Reply:
yes, Done